packages/ui/src/components/Banner/index.tsx

```diff
@@ -0,0 +1,119 @@
+import styled from '@emotion/styled'
+import type { ComponentProps, ReactNode } from 'react'
+import { ButtonV2 } from '../ButtonV2'
+import { Link } from '../Link'
+import { Stack } from '../Stack'
+import { Text } from '../Text'
+
+type Type = 'intro' | 'promotional'
+type Size = 'small' | 'medium'
+
+const Container = styled('div', {
+  shouldForwardProp: prop => !['type', 'size'].includes(prop),
+})<{ type: Type; size: Size }>`
+  padding: ${({ theme, size }) => theme.space[size === 'small' ? '2' : '3']};
+  border-radius: ${({ theme }) => theme.radii.large};
+  background: ${({ theme, type }) =>
+    type === 'intro'
+      ? theme.colors.primary.background
+      : theme.colors.primary.backgroundStrong};
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: ${({ theme }) => theme.space['2.25']};
+
+  > svg:first-child,
+  > img {
+    height: ${({ size }) => (size === 'medium' ? '140px' : '100px')};
+    align-self: center;
+  }
+`
+
+type BannerProps = {
+  type?: Type
+  size?: 'small' | 'medium'
+  title: string
+  children: ReactNode
+  direction?: 'row' | 'column'
+  onClose?: ComponentProps<typeof ButtonV2>['onClick']
+  buttonText?: string
+  onClickButton?: ComponentProps<typeof ButtonV2>['onClick']
+  linkText?: string
+  linkHref?: string
+  image?: ReactNode
+  className?: string
+}
+
+export const Banner = ({
+  type = 'intro',
+  size = 'medium',
+  title,
+  children,
+  direction = 'column',
+  onClose,
+  buttonText,
+  onClickButton,
+  linkText,
+  linkHref,
+  image,
+  className,
+}: BannerProps) => (
+  <Container type={type} size={size} className={className}>
+    {image}
+    <Stack
```

Comment:
Could you integrate an internal state so when we click on close button the banner will close even if the developper didn't managed the state. You can check Alert to see how it is already done :)

Reply:
Done!